tests/dataframe/test_drop_pytest.py

```diff
@@ -50,3 +57,30 @@ def test_drop_all_index(self, df):
         ):
             assert dropped.shape == (0, cols)
             assert list(dropped.to_pandas().index) == []
+
+    def test_drop_raises(self):
+        ed_flights = self.ed_flights()
+
+        with pytest.raises(
```

Comment:
You need to have a separate `with pytest.raises(...)` for every statement you're expecting an error from.